cmd/goodman/main.go

```diff
@@ -51,7 +51,7 @@ func main() {
 				}
 			}()
 			for retries := 5; retries > 0; retries-- {
-				runner, err := goodman.NewRunner("Hooks", hookServerInitalPort)
+				runner, err := goodman.NewRunner("HooksRunner", hookServerInitalPort)
```

Comment:
😢 for magic strings.  I wonder if its possible to look this up using reflection or some other mechanism?